### ISSUE TITLE
RPM Brackets needed for Fedora and OpenSuse

### ISSUE THREAD

**Issue by jalalsfs** [adobe/brackets#14011]
Hello there!
There was two or more projects to provide PRM packages for Fedora community, but all have being stopped.
From the developer of Brackets can we expect RPM file for Fedora soon, and many other similar projects (VSCode, Sublime ...)
Thank you.

**Comment by nethip** [comment]
@jalalsfs I agree we aspire to have installers for Fedora community. But unfortunately we don't have that kind of bandwidth to work on creation and support of RPM packages as Linux Brackets is mostly community driven.

**Comment by jalalsfs** [comment]
@nethip If there is a chance to start up RPM with the help of the guys who already packaged Brackets for Fedora in corp like @sachinkama and others it will be easy to move on towards make it real.

**Comment by ingmarcosp** [comment]
Hello! for Opensuse 42.3 there  are package, but they are outdated. It will be nice have updated packages!

**Comment by RustyRaptor** [comment]
@nethip  Why don't we just retire the .deb package and go with something that will work on all distros like appimage? It makes so much more sense for such a widely used application and it has worked so well for other projects.

**Comment by jalalsfs** [comment]
@RustyRaptor The advantage of getting RPM along with DEB package in the official Brackets' package repositories is the updating from your package manager easily.
See VSCode or google-chrome, just updating your system and everything's done.

**Comment by RustyRaptor** [comment]
@jalalsfs  I understand this however that only applies to appimages. Using Snappy and Flatpak you can also update through the package manager or even the GUI.

**Comment by nethip** [comment]
@RustyRaptor I am pinging @rajnishb for his inputs.

**Comment by rexfordnyrk** [comment]
Hello any progress on this?? I am thinking forking and providing at least .rpm files for fedora x86_64 and i386 as this are the only ones I have time to support for now.

**Comment by shubhsnov** [comment]
@rexfordnyrk I don't think we're working on .rpm packages yet but it would be a great thing to have. All community help is appreciated.

**Comment by rexfordnyrk** [comment]
@shubhsnov  ok great! I have already compiled the current release both i386 and x86_64(which I am personally using atm). All I gotta do is fork, create a brunch or something and upload them. or are there any guidelines to follow? let me know. I will put that on my todo list for Monday when I get to the office, my bandwidth at home ain't that good for uploads.Cheers

**Comment by rexfordnyrk** [comment]
@shubhsnov @jalalsfs  @nethip  
So I did as you said and forked the repo. I Published a release based on the TAG release 1.14 and uploaded the RPM binaries. it can be found here 
[here](https://github.com/rexfordnyrk/brackets/releases/tag/release-1.14)

**Comment by shubhsnov** [comment]
@rexfordnyrk Thanks.
Hope this helps the community.
